python-package/lightgbm/sklearn.py

```diff
@@ -919,9 +919,10 @@ def predict_proba(self, X, raw_score=False, start_iteration=0, num_iteration=Non
         """Docstring is set after definition, using a template."""
         result = super().predict(X, raw_score, start_iteration, num_iteration, pred_leaf, pred_contrib, **kwargs)
         if callable(self._objective) and not (raw_score or pred_leaf or pred_contrib):
-            _log_warning("Cannot compute class probabilities or labels "
-                         "due to the usage of customized objective function.\n"
-                         "Returning raw scores instead.")
+            new_line = "\n"
+            _log_warning(f"Cannot compute class probabilities or labels "
+                         f"due to the usage of customized objective function.{new_line}"
+                         f"Returning raw scores instead.")
```

Comment:
```suggestion
            _log_warning("Cannot compute class probabilities or labels "
                         "due to the usage of customized objective function.\n"
                         "Returning raw scores instead.")
```

It looks like the unnecessary `new_line` variable from https://github.com/microsoft/LightGBM/pull/4188/files#r615043256 was added back. I think it can be removed, and strings without any templating do not need an `f`.